jsystem/requirements.fasttext_clf.txt

```diff
@@ -0,0 +1 @@
+flask==2.0.2
```

Comment:
This is the only python package dependency? What about the fasttext module?

Reply:
The fasttext package is installed from GitHub in the Dockerfile.